cms/templates/activation_complete.html

```diff
@@ -9,8 +9,8 @@
 <div class="wrapper-mast wrapper sr">
   <header class="mast">
     <h1 class="page-header">
-      ${_("{studio_name} Account Activation").format(
-        studio_name=Text(settings.STUDIO_SHORT_NAME)
+      ${Text(_("{studio_name} Account Activation")).format(
+        studio_name=Text(settings.STUDIO_SHORT_NAME),
```

Comment:
You are correct here. I didn't pay close enough attention. The last part of this PR has a change with HTML(), and there you must use Text().